blockchain/src/main/java/com/tangem/blockchain/blockchains/ethereum/EthereumUtils.kt

```diff
@@ -428,6 +428,12 @@ class EthereumUtils {
             return CompiledEthereumTransaction(transaction, hash)
         }
 
+        fun createErc20ApproveData(spender: String, amount: Amount?): String =
```

Comment:
Думаю, стоит уточнить, что это hex:
```suggestion
        fun createErc20ApproveDataHex(spender: String, amount: Amount?): String =
```